lib/logstash/filters/util/db_object.rb

```diff
@@ -0,0 +1,136 @@
+require_relative "validatable"
+require_relative "column"
+
+module LogStash module Filters module Util
+
+  TEMP_TABLE_PREFIX = "temp_".freeze
+
+  class DbObject < Validatable
```

Comment:
Discussed face 2 face, mano a mano like - we decided to keep the validations in the same class that used the validated options, less chance of drift i.e. the main class using a new option that the validation class was not updated for.